Ginger/GingerCoreNET/Database/NoSqlBase/GingerHbase.cs

```diff
@@ -577,6 +595,10 @@ public static string DisplayInferredTypeAndValue(byte[] byteArray)
             {
                 return "";
             }
+            if (byteArray.All(y => y == 0))
```

Comment:
Add condition to check if bytearray length is less than 9, then only check for all 0

Reply:
done